setup.cfg

```diff
@@ -19,12 +19,14 @@ packages = find:
 install_requires =
     certifi
     chardet
+    docker
```

Comment:
these dependencies are only for tests. Please move them in the `extras_require` test section